module3/labs/Week4/UpdatingResources.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: page
+title: Updating Resources in MVC Lab
+---
+
+For today's lab, we are going to continue building the Record Collection application.
+
+**Objective**
+* Add a form for updating Album
+* Add a form for updating Songs
+
+
+## Setup
+* Open your Record Collection project.
+    * Run the project, and make sure you have the Record#Index, #Show, #New, and #Create actions working.
+* If you have been absent or can't find your project, follow these steps:
+    * Fork and Clone it from this [Starter Repository](https://github.com/turingschool-examples/LaunchFeatureLabStarter_RecordCollection).
+    * Build the project (let your instructor know if you receive any errors!).
+        * Open the database in pgAdmin, and create some records by running the following script (feel free to update the albums and artists to your favorites!).
+    ```sql
+    INSERT INTO albums (title, artist, release_date)
+    VALUES
+    ('Stop Making Sense', 'Talking Heads', '1984-06-24'),
+    ('Banana Winds', 'Jimmy Buffet', '1996-12-06')
+    ```
+    * Check out the branch _____________________ TODO need a branch ___________
```

Comment:
Figuring out what we're doing for branches for the labs that build off of each other, still feels like an open question. 

For the sake of not building more than we need to, I'm thinking maybe we start by having them build off of their own code from the day before. And if someone's code is getting messy or someone misses a day we can make the needed branch then. And maybe even pull that code from one of the student's work?

Reply:
Yea, I think we talked about us building along with them during lab so we have a fresh branch for the next day.